### PR TITLE
Add CSSFontFaceRule and CSSImportRule APIs

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds BCD data for two CSS rule APIs: the font face rule, and the import rule.  Versions were determined using the mdn-bcd-collector project, along with a slight bit of mirroring (Firefox -> Fenix, Chrome -> CrA, Opera, Opera Android, Samsung Internet, WebView).

These two interfaces are a part of DOM Level 2:
https://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSSFontFaceRule
https://www.w3.org/TR/DOM-Level-2-Style/css.html#CSS-CSSImportRule

### CODE REVIEW

File: api/CSSFontFaceRule.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
+  "api": {
+    "CSSFontFaceRule": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSFontFaceRule",
```
Comment (foolip) [inline api/CSSFontFaceRule.json:5]: The current spec for this is https://drafts.csswg.org/css-fonts/#om-fontface, and it matches this, just the interface with a `style` property.

File: api/CSSImportRule.json [hunk 2]
```diff
@@ -0,0 +1,196 @@
+{
+  "api": {
+    "CSSImportRule": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CSSImportRule",
```
Comment (foolip) [inline api/CSSImportRule.json:5]: This is spec'd in https://drafts.csswg.org/cssom/#the-cssimportrule-interface, and that matches the subfeatures being added here.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@vinyldarkscratch in order to figure out which spec something is in, I would suggest not looking at https://www.w3.org/TR/, but instead search in https://github.com/w3c/webref/tree/master/ed/idl. The most up-to-date version of the spec is linked at the top of each file there.

@ddbeck is there anything special one should do when adding an entry?

**Comment by ddbeck** [comment]
@foolip 

> is there anything special one should do when adding an entry?

No, unlike removals, I haven't been doing anything special for new entries. Let me know if you think I _should_ though. 😆

**Comment by foolip** [comment]
@ddbeck alright, in that case I will just merge away without the release notes label :)

**Pull request merged**